test/checkValidToken.test.js

```diff
@@ -0,0 +1,37 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../src/index';
+
+const { expect } = chai;
+chai.use(chaiHttp);
+
+describe('testing the signout controller', () => {
+  it('should check for an empty token and alert that the user is unauthorized', (done) => {
+    chai
+      .request(app)
+      .get('/api/auth/signout')
```

Comment:
let's change this route to match our readme

Reply:
yes lemme work on it